subsys/nrf_security/src/drivers/cracen/cracenpsa/include/cracen_psa_kmu.h

```diff
@@ -55,16 +55,29 @@ enum cracen_kmu_metadata_key_usage_scheme {
 	CRACEN_KMU_KEY_USAGE_SCHEME_RAW
 };
 
-/**
- * @brief Retrieves the slot number for a given key handle.
+/** Retrieves the slot number for a given key handle.
  *
  * @param[in]  key_id      Key handler.
  * @param[out] lifetime    Lifetime for key.
  * @param[out] slot_number The key's slot number.
- *
- * @return psa_status_t
```

Comment:
Yes but what information does that add? It's very clear from the function prototype what type it returns. It would be useful information if it listed the actual error codes, but the way it was it's not.